packages/teleport-project-packer/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@teleporthq/teleport-project-packer",
+  "version": "0.6.0",
+  "description": "Teleport packer",
+  "author": "teleportHQ",
+  "license": "MIT",
+  "homepage": "https://teleporthq.io/",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/teleporthq/teleport-code-generators.git"
+  },
+  "bugs": {
+    "url": "https://github.com/teleporthq/teleport-code-generators/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc"
+  },
+  "dependencies": {
+    "@teleporthq/teleport-generator-shared": "^0.6.0",
+    "@teleporthq/teleport-project-generator-react-basic": "^0.6.0",
```

Comment:
is the generic packer using the generators and the publishers as direct references? or is this just temporary until we figure out the interface?

Reply:
It's only temporary. I will remove the dependencies from the package.json file